app/controllers/collections_controller.rb

```diff
@@ -129,7 +129,7 @@ def loss_report
         format.json { render json: JSON.pretty_generate(compared_collections) }
       else
         format.text { render plain: loss_report_output(compared_collections: compared_collections, hide_items: true, display: 'text') }
-        format.json { render json: JSON.pretty_generate(loss_report_output(compared_collections: compared_collections, hide_items: false, display: 'json')) }
```

Comment:
Doesn't removing this mean that all paths are using the default behavior of true? If so, why have the option?

Reply:
The option is only for `text_output`, it doesn't affect `json_output`

Reply:
I suppose then, to answer your question, the purpose of having the option (which only affects `text_output`) is for debugging if a dev is using the `text_output` option.

Reply:
I think I would prefer it to be pulled out of the params with a default value then.  E.g. url/loss_report?hide_items=false should work